memory.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
The file looks good but as mentioned in #10 it should not be a part of this repository.